README.md

```diff
@@ -106,13 +106,10 @@ fastify.register(require('fastify-postgres'), {
 })
 
 fastify.post('/user/:username', (req, reply) => {
```

Comment:
If we make this `async`, we can directly return `return client.query` right?

Reply:
There is no need to use async at all because Fastify understand promises. However that won't be the normal case for applications.

Reply:
Ah yes, thank you for the info!